frontend/packages/component-library/src/cms/header/config.ts

```diff
@@ -130,6 +123,13 @@ export function getDefaultHeaderProps({
         image: pythonLabImage,
         description: 'Code using Python',
       },
+      {
```

Comment:
@kelbyhawn could you check to see if this is displayed correctly in contentful? Thank you!

Reply:
Looks great, thanks for updating! Screenshots for confirmation: 

| Storybook | Contentful | 
| ---- | ---- |
| <img width="305" alt="Screenshot 2025-05-21 at 3 22 22 PM" src="https://github.com/user-attachments/assets/51846256-b6fa-4ff6-8eeb-6173d3292b20" /> | <img width="275" alt="Screenshot 2025-05-21 at 3 22 32 PM" src="https://github.com/user-attachments/assets/fd78b236-c8ab-4231-95b0-e6e021b43b6e" /> |

Reply:
Thanks for checking!